e2e/test-initcontainer-volumes.yaml

```
@@ -0,0 +1,27 @@
+---
+apiVersion: k6.io/v1alpha1
+kind: K6
+metadata:
+  name: k6-sample-init-container-volume
+spec:
+  parallelism: 1
+  script:
+    localFile: /test/test.js
+  runner:
+    volumes:
+      - emptyDir: {}
+        name: k6-provision-location
+    volumeMounts:
+      - mountPath: /test
+        name: k6-provision-location
+    initContainers:
+      - image: busybox:1.28
+        command:
+          [
+            "sh",
+            "-c",
+            "cd /test; wget https://github.com/grafana/k6-operator/blob/main/e2e/test.js;",
```

```suggestion
            "cd /test; wget https://raw.githubusercontent.com/grafana/k6-operator/main/e2e/test.js;",
```
I think this link needs to be changed. Right now this sample test fails with:
```
time="2023-08-02T11:38:48Z" level=error msg="SyntaxError: file:///test/test.js: Unexpected token (7:0)\n   5 | \n   6 | \n>  7 | <!DOCTYPE html>\n     | ^\n   8 | <html lang=\"en\" data-color-mode=\"auto\" data-light-theme=\"light\" data-dark-theme=\"dark\"  data-a11y-animated-images=\"system\" data-a11y-link-underlines=\"false\">\n   9 |   <head>\n  10 |     <meta charset=\"utf-8\">\n\tat <internal/k6/compiler/lib/babel.min.js>:2:28536(99)\n\tat <internal/k6/compiler/lib/babel.min.js>:14:24413(11)\n\tat bound  (native)\n\tat s (<internal/k6/compiler/lib/babel.min.js>:1:1327(8))\n" hint="script exception"
```
So it downloads HTML instead of JS.